pkgs/development/libraries/libpostal/default.nix

```diff
@@ -11,11 +11,43 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-gQTD2LQibaB2TK0SbzoILAljAGExURvDcF3C/TfDXqk=";
   };
 
+  # languageClassifier, libpostalData and parser were not release for libpostal 1.1, using data from 1.0 instead.
+  languageClassifier = fetchzip {
+    url = "https://github.com/openvenues/libpostal/releases/download/v1.0.0/language_classifier.tar.gz";
+    sha256 = "sha256-/Gn931Nx4UDBaiFUgGqC/NJUIKQ5aZT/+OYSlcfXva8=";
+    stripRoot = false;
+  };
+
+  libpostalData = fetchzip {
+    url = "https://github.com/openvenues/libpostal/releases/download/v1.0.0/libpostal_data.tar.gz";
+    sha256 = "sha256-FpGCkkRhVzyr08YcO0/iixxw0RK+3Of0sv/DH3GbbME=";
+    stripRoot = false;
+  };
+
+  parser = fetchzip {
+    url = "https://github.com/openvenues/libpostal/releases/download/v1.0.0/parser.tar.gz";
+    sha256 = "sha256-OHETb3e0GtVS2b4DgklKDlrE/8gxF7XZ3FwmCTqZbqQ=";
+    stripRoot = false;
+  };
+
+  data = stdenv.mkDerivation {
+    name = "libpostal-data";
+    buildCommand = ''
+      mkdir -p $out/data/libpostal
+
+      ln -s ${languageClassifier}/language_classifier   $out/data/libpostal/languageClassifier
+      ln -s ${libpostalData}/transliteration            $out/data/libpostal/transliteration
+      ln -s ${libpostalData}/numex                      $out/data/libpostal/numex
+      ln -s ${libpostalData}/address_expansions         $out/data/libpostal/address_expansions
+      ln -s ${parser}/address_parser                    $out/data/libpostal/address_parser
+    '';
+  };
+
```

Comment:
This should all go in a `let … in` block before `mkDerivation`. If you bind it inside of `mkDerivation`, it gets made available at build time even if not used, and so the data ends up unconditionally downloaded.